src/test/ui/simd-intrinsic/simd-intrinsic-generic-select.rs

```diff
@@ -52,5 +53,14 @@ fn main() {
 
         simd_select(z, z, z);
         //~^ ERROR mask element type is `f32`, expected `i_`
+
+        simd_select_bitmask(0u8, x, x);
+        //~^ ERROR mask length `8` != other vector length `4`
+
+        simd_select_bitmask(0.0f32, x, x);
+        //~^ ERROR `f32` is not an integral type
+
+        simd_select_bitmask("x", x, x);
+        //~^ ERROR `&str` is not an integral type
```

Comment:
Might want to add a test for when the vector is not a SIMD type (probably for `simd_select` as well).

Reply:
I think `#[repr(simd)]` is gating that, right?

Reply:
I think @gnzlbg means the case when the second/third argument is not a `repr(simd)` type. That would exercise the `require_simd!` check in the intrinsic implementation.

Reply:
Ah indeed, tests added now!